docs/develop-logseq-on-windows.md

```diff
@@ -1,73 +1,43 @@
 # Build Logseq Desktop on Windows
 
-## Intro
-This is a guide on creating Logseq development environment on Windows with `PowerShell`. Non-platform specific instructions like [Develop Logseq](develop-logseq.md) **should also be referenced**.
+This is a guide on setting up a Logseq development environment on Windows.
 
 ## Pre-requisites
-* Ensure `Set-ExecutionPolicy Unrestricted` (or other equivalent)
-* Good network connection. Here's [An example of setting up proxy in PowerShell](#an-example-of-setting-up-proxy-in-powershell)
-* Node.js 18.x
-* Clojure (follow this [Guidance](https://clojure.org/guides/getting_started#_installation_on_windows))
-* JRE 8 (required for Clojure)
-* Visual Studio (required for desktop app)
 
-(updated 20230221. May confirm via JAVA_VERSION and NODE_VERSION in [THIS FILE](https://github.com/logseq/logseq/blob/master/.github/workflows/build.yml))
-
-### An example of installing pre-requisites on Windows
-* Install [Chocolatey](https://chocolatey.org/)
-* Install JRE (only x64, x32 will not work)
-* Install NVM for Windows, Node.js, and Yarn
+  * NVM, Node, and Yarn
   ```
-  choco install nvm
-  nvm install 18.12.0 (or whatever version)
-  nvm use 18.12.0
+  winget install --id CoreyButler.NVMforWindows
+  nvm install 18
+  nvm use 18
   npm install -g yarn
-  nvm use 18.12.0
   ```
-* Install [clj-on-windows](https://github.com/clojure/tools.deps.alpha/wiki/clj-on-Windows)
-
-Congrats! The pre-requisites are ready.
-
-## Set-up development environment (web app)
-
-The basic idea is replacing the `clojure` commands in [package.json](https://github.com/logseq/logseq/blob/master/package.json) to `clj`.
-Go to your cloned Logseq repo. Then install dependencies, execute the `clj` equivalent of `yarn watch`. Refer [THIS](#an-example-of-setting-up-proxy-in-powershell) if you want to setup proxy in `PowerShell`.
-
-* Copy files in `resources` to `static`
-
-* Compile static assets(css, icons...)
+  * Java
   ```
-  yarn add --dev gulp
-  yarn
-  yarn gulp:watch
+  winget install --id Microsoft.OpenJDK.17
   ```
+  * Visual Studio
+  ```
+  winget install --id Microsoft.VisualStudio.2022.Community
+  ```
+  * An installer for clojure is available from [casselc/clj-msi](https://github.com/casselc/clj-msi/releases/)
 
-* Open another powershell window, and run `yarn cljs:watch`. Clojure CLI will pull dependencies from Maven and Clojars, build the app and start the development server. Refer [THIS](#set-up-clojure-cli-repository-mirror) if your network access to Maven and Clojars is unstable.
```

Comment:
Shouldn't be deleted.

Reply:
The Windows doc was inconsistent and (I believe?) out of date w/r/t the main develop-logseq.md doc which says to use `yarn watch`, and as a new contributor I was pretty confused which one to follow.  Rather than trying to keep them in sync, I think this doc should just focus on windows-specific setup and then the regular development path is in develop-logseq.md.

Reply:
The mirror instructions are moved to a troubleshooting section, IMO that's not something most people will need to think about.

Reply:
Thanks for the clearification.